docs/EN_US/ECLWatch/ECLWa_mods/SessFaq.xml

```diff
@@ -0,0 +1,250 @@
+        out. Your session will get locked. Anything typed into any fields
+        (such as a search box) that has not been submitted or entered could
+        potentially be lost. However, since the session is only locked, it is
+        unlikely that any data will be lost.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will my queued and
+        scheduled workunits run when I am locked out?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, the session only
+        applies to ESP/ECL IDE and ESP/ECLWatch communications.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will HPCC command line
+        utilities be affected?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Possibly. If you have
+        configured AuthPerSessionOnly then some command line utilities may not
+        function the same as before session management.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Does auto refresh in
+        ECLWatch reset the session expiration timer?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No. Only active
+        interactions like mouse clicks and pressing keys extend the timeout.
+        Note that scrolling does not extend the expiration timer.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: If I am logged in to the
+        same account using multiple tabs in a browser, or multiple instances
+        of the same browser, can I get logged out of one but not the
+        others?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No, activity is tracked by
+        your credentials. Activity in one tab or instance extends the session
+        for all.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: If I am logged in to the
+        same account using different browsers (e.g., Firefox and Chrome), do
+        they share the same session timeout?</para>
+
+        <para><emphasis role="strong">A</emphasis>: No. Since each browser has
+        its own cookie store, activity in one does not extend to the
+        other.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Can I automatically return
+        to the ECLWatch screen where I was when automatically locked
+        out?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes. The intent is to lock
+        your session and not completely log you out. Therefore logging back in
+        should return to the same point when your session locked.</para>
+      </listitem>
+
+      <listitem>
+        <para><emphasis role="strong">Q</emphasis>: Will I be able to change
+        an expired password from the login screen?</para>
+
+        <para><emphasis role="strong">A</emphasis>: Yes, this behaves just as
```

Comment:
If your password is expired, you are redirected to a page that allows you to reset it